dotCMS/src/test/java/com/dotmarketing/business/cache/provider/CacheSizingUtilTest.java

```diff
@@ -0,0 +1,57 @@
+package com.dotmarketing.business.cache.provider;
+
+import org.junit.Test;
+import com.dotcms.repackage.com.google.common.base.Optional;
+import com.dotmarketing.portlets.contentlet.business.Contentlet;
+
+public class CacheSizingUtilTest {
+
+    
+    final String fourLetters = "CRAP";
+
+    
+    CacheSizingUtil cacheSizer = new CacheSizingUtil();
+    
+    @Test
+    public void test_sizeof_works() {
+       
+        final Contentlet con = new Contentlet();
+        
+        long contentletSize = cacheSizer.sizeOf(con);
+        assert(contentletSize>0);
+        
+        
+
+        
+        long fourLettersSize = cacheSizer.sizeOf(fourLetters);
+        assert(fourLettersSize==48);
+        
+        
+    }
+    
+    @Test
+    public void test_sizeof_null_works() {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6315945)